rideshare_worksheet.rb

```diff
@@ -0,0 +1,43 @@
+rides_by_driver =
+  [
+    [{:driver_id => "DR0001", :rider => "RD0003", :date => "3rd Feb 2016", :cost => 10, :rating => 3},
+     {:driver_id => "DR0001", :rider => "RD0015", :date => "3rd Feb 2016", :cost => 30, :rating => 4},
```

Comment:
I like the idea here of organizing trips by driver. However, rather than using an array as the outermost layer, it might be more clear to use a hash, where the keys are driver IDs, for a few reasons:
- It removes the repeated information in the inner-most hashes
- It would make your loops below a small bit simpler (you would get the driver ID as a loop parameter rather than having to go find it)